diff --git a/Makefile b/Makefile
index 34b050e..2bb65aa 100644
--- a/Makefile
+++ b/Makefile
@@ -541,6 +541,31 @@ else ifeq ($(platform), miyoo)
 	USE_CYCLONE := 1
 	USE_DRZ80 := 1
 
+# miyoo a30
+else ifeq ($(platform), r36s)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR = $(CROSS_COMPILE)ar
+	fpic := -fPIC
+	LDFLAGS += $(fpic) -shared  -Wl,-z,noexecstack -Wl,--version-script=link.T -Wl,-no-undefined
+	CFLAGS += -Ofast \
+	-flto=4 -fwhole-program -fuse-linker-plugin \
+	-fdata-sections -ffunction-sections -Wl,--gc-sections \
+	-fno-stack-protector -fno-ident -fomit-frame-pointer \
+	-falign-functions=1 -falign-jumps=1 -falign-loops=1 \
+	-fno-unwind-tables -fno-asynchronous-unwind-tables -fno-unroll-loops \
+	-fmerge-all-constants -fno-math-errno \
+	$(MYARCH)
+	CXXFLAGS += $(CFLAGS)
+	CPPFLAGS += $(CFLAGS)
+	ASFLAGS += $(CFLAGS)
+	HAVE_NEON = 1
+	ARCH = arm
+	CPU_ARCH := arm
+	ARM = 1
+
+
 # Emscripten
 else ifeq ($(platform), emscripten)
 	TARGET := $(TARGET_NAME)_libretro_$(platform).bc
