diff --git a/Makefile.libretro b/Makefile.libretro
index a1c8c7a..641a9a7 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -27,7 +27,7 @@ AS ?= as
 CC_AS ?= $(CC)
 CFLAGS ?=
 
-TARGET_NAME := km_pcsxtreme_hd
+TARGET_NAME := pcsx_rearmed
 ARCH_DETECTED := $(shell $(CC) -dumpmachine | awk -F- '{print $$1}')
 GIT_VERSION := " $(shell git rev-parse --short HEAD || echo unknown)"
 ifneq ($(GIT_VERSION)," unknown")
@@ -491,6 +491,25 @@ else ifeq ($(platform), miyoo)
 	DYNAREC = ari64
 	HAVE_NEON = 0
 
+# miyoo a30
+else ifeq ($(platform), r36s)
+   TARGET := $(TARGET_NAME)_libretro.so
+   CC = $(CROSS_COMPILE)gcc
+   CXX = $(CROSS_COMPILE)g++
+   AR = $(CROSS_COMPILE)ar
+   
+   CFLAGS += $(MYARCH) -fomit-frame-pointer
+   CFLAGS += -ffast-math -fdata-sections -ffunction-sections -fsingle-precision-constant -flto -fPIC
+   LDFLAGS += -flto -fPIC  -Wl,-z,noexecstack 
+   LDFLAGS += -flto
+   CPU_ARCH := arm
+   OPTIMIZE += -Ofast -DNDEBUG=1
+   HAVE_NEON = 1
+   HAVE_NEON_ASM = 1
+   BUILTIN_GPU = neon
+   ARCH = arm
+   DYNAREC = ari64
+
 # Emscripten
 else ifeq ($(platform), emscripten)
    TARGET  := $(TARGET_NAME)_libretro_$(platform).bc
