diff --git a/Makefile b/Makefile
index dc4b60d..45fce6f 100644
--- a/Makefile
+++ b/Makefile
@@ -524,6 +524,25 @@ else ifeq ($(platform), miyoomini)
 	ARCH = arm
 	BUILTIN_GPU = neon
 
+# miyoo a30
+else ifeq ($(platform), r36s)
+   TARGET := $(TARGET_NAME)_libretro.so
+   CC = $(CROSS_COMPILE)gcc
+   CXX = $(CROSS_COMPILE)g++
+   AR = $(CROSS_COMPILE)ar
+   SHARED := -shared  -Wl,-z,noexecstack -Wl,--version-script=link.T
+   fpic := -fPIC -DPIC
+   LDFLAGS += -fPIC -flto
+   PLATFORM_DEFINES := -DCC_RESAMPLER -DCC_RESAMPLER_NO_HIGHPASS
+   CFLAGS += $(MYARCH) -fomit-frame-pointer
+   CFLAGS += -fomit-frame-pointer -ffast-math -fPIC -flto
+   HAVE_NEON = 1
+   ARCH = arm
+   BUILTIN_GPU = neon
+   CPU_ARCH := arm
+   MMAP_JIT_CACHE = 1
+   HAVE_DYNAREC := 1
+
 # Windows
 else
 	TARGET := $(TARGET_NAME)_libretro.dll
